src/main/java/MAYGEN/MAYGEN.java

```diff
@@ -1925,29 +1933,35 @@ public static boolean allIs0(int[] list) {
      *
      * @param A int[][] adjacency matrix
      * @return boolean
-     * @throws IOException 
+     * @throws IOException
      */
-    
     public static int indexYZ() {
-    	int index=0;    	
-    	for(int i=0;i<=r;i++) {
-    		index=index+initialPartition[i];
-    	}
-    	return index-1;
+        int index = 0;
+        for (int i = 0; i <= r; i++) {
+            index = index + initialPartition[i];
+        }
+        return index - 1;
     }
+
     public static boolean canonicalTest(int[][] A) throws IOException {
         boolean check = true;
         learningFromCanonicalTest = false;
-        int value=indexYZ();
+        int value = indexYZ();
         y = ys[value];
         z = zs[value];
         if (partSize == r && z != 1) {
             z = z - 1;
         }
-        clearFormers(false,y);
+        clearFormers(false, y);
         boolean test = true;
         for (int i = y; i <= z; i++) {
-            test = rowCanonicalTest(i, r, A, partitionList.get(i), canonicalPartition(i, partitionList.get(i)));
+            test =
+                    rowCanonicalTest(
+                            i,
+                            r,
+                            A,
+                            partitionList.get(i),
+                            canonicalPartition(i, partitionList.get(i)));

```

Comment:
It was from formatting plugin

Reply:
@javadev so the formatting plugin puts all the parameters line by line separately ?